### DISCUSSION TITLE
Is it possible using laravel-html to create link that includes onclick js?

### DISCUSSION THREAD

**Discussion by levaconline** [spatie/laravel-html#201] [Q&A] [answered]
Is it possible using laravel-html to create link that includes onclick js, like following (in laravel blades):
`<a class="btn btn-outline btn-danger btn-xs" title="{{ trans('common.delete') }}" href="{!! route('storage-ingests.delete', [$storageIngest->id]) !!}" onclick="return confirm('{{ trans('modules/storage_ingests.confirm_deletion') }}')"><i class="fa fa-trash-o"></i></a>`

**Comment by sebastiandedeyne** [comment, marked as answer]
I believe you can use the generic `attribute` method.

```php
html()->a()->attribute('onclick', 'return confirm("' . trans('modules/storage_ingests.confirm_deletion') . '")');
```

**Reply by levaconline** [reply]
In some cases elements can contain multiple and more complex attributes. (especially combination of multiselect with Select2 JS library). But your answer works!
Thanks a lot for help!
